test/vectors/presentation/4.present-proof_ack.json

```diff
@@ -0,0 +1,9 @@
+{
+   "from":"did:peer:1zQmT1xfJgfCyXoUGQ7PLxDrBrcVp33MgvDXmP6uVj2jnX8m",
+   "id":"63e87069-8dee-4754-9be8-56b22901dfef",
+   "thid":"aad8e13a-01dc-4541-a961-70f1754dc5f1",
+   "type":"https://didcomm.org/present-proof/3.0/ack",
+   "body":{
+      
```

Comment:
empty body intentional?

Reply:
Its mandatory in DIDComm v2, even if it's empty - https://identity.foundation/didcomm-messaging/spec/#message-headers.

> body - REQUIRED. The body attribute contains all the message type specific attributes of the message type indicated in the type attribute. This attribute MUST be present, even if empty. It MUST be a JSON object conforming to [RFC 7159](https://datatracker.ietf.org/doc/html/rfc7159).